main.go

```diff
@@ -1,143 +0,0 @@
-/*
```

Comment:
Why are we making `main.go` a symlink?  I don't see any advantage to doing so.

Reply:
to maintain compatibility with operator-sdk